pandas/core/generic.py

```diff
@@ -2063,18 +2063,73 @@ def __delitem__(self, key):
 
     def take(self, indices, axis=0, convert=True, is_copy=True, **kwargs):
         """
-        Analogous to ndarray.take
+        Return the elements in the given *positional* indices along an axis.
+
+        This means that we are not indexing according to actual values in
+        the index attribute of the object. We are indexing according to the
+        actual position of the element in the object's array of values.
```

Comment:
I would just write "actual position of the elements in the object" 
("object's array of values" can possibly cause some confusion I think)